Gateway/Request/APMBuilder.php

```diff
@@ -233,6 +234,12 @@ public function build(array $buildSubject)
                     self::PLATFORM_TYPE => $this->helper->getPlatformType(),
                 ];
                 break;
+            case Grabpay::CODE:
+                $paymentInfo[self::SOURCE] = [
+                    self::SOURCE_TYPE => 'grabpay',
+                    self::PLATFORM_TYPE => $this->helper->getPlatformType(),
```

Comment:
Does it use a platform type? 🤔

Reply:
I figured there's no harm in sending it (maybe the rms implementation might need it). and it looks like most other APMs are sending this field through too

Reply:
Can confirm about rms -> yep will need to send `PLATFORM_TYPE` to RMS channel